ext/LuxSimpleChainsExt.jl

```diff
@@ -66,8 +66,8 @@ end
 
 __to_simplechains_adaptor(layer) = throw(SimpleChainsModelConversionError(layer))
 
-function Lux.initialparameters(::AbstractRNG, layer::SimpleChainsLayer)
-    return (; params=Array(SimpleChains.init_params(layer.layer)))
+function Lux.initialparameters(rng::AbstractRNG, layer::SimpleChainsLayer)
+    return (; params=Array(SimpleChains.init_params(layer.layer; rng=rng)))
```

Comment:
```suggestion
    return (; params=Array(SimpleChains.init_params(layer.layer; rng)))
```

Reply:
...still figuring out the nuances to the kwarg syntax in modern Julia. Thanks.